mail_activity_board/views/mail_activity_view.xml

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--
+    VIEWS
+-->
+
+    <!-- FORM POP
+
+        Add a link to the related model instance to ease navigation
+    -->
+    <record id="mail_activity_view_form_popup" model="ir.ui.view">
+        <field
+            name="name"
+        >mail.activity.view.form.popup (in mail_activity_board)</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_form_popup" />
+        <field name="arch" type="xml">
+            <field name="activity_type_id" position="before">
+                <field
+                    name="related_model_instance"
+                    invisible="not related_model_instance"
+                />
+            </field>
+        </field>
+    </record>
+
+    <!-- CALENDAR VIEW
+        Add a link to the related model instance to ease navigation
+    -->
+    <record id="mail_activity_view_calendar" model="ir.ui.view">
+        <field
+            name="name"
+        >mail.activity.view.form.calendar (in mail_activity_board)</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_calendar" />
+        <field name="arch" type="xml">
+            <field name="res_name" position="before">
+                <field name="related_model_instance" />
```

Comment:
If I click on an activity in the calendar view, there is a problem rendering the widget for this field.
```
Error: The following error occurred in onWillStart: "value.id is undefined"
```
![image](https://github.com/user-attachments/assets/cbf8df73-3c7c-4a03-a6db-458547e18f26)

To be fair, this is not a migration issue. The same issue happens in 17.0. But it would still be good to have this fixed. Maybe we can render the record's `res_id` instead?